docs/tests.md

```diff
@@ -141,3 +141,41 @@ the system tests.
 1. `DEBUG_SPECTRON=1 npm run test:system-tests`.
 1. Use the "Attach to Process (Inspector Protocol)" debug configuration to
    attach to the process (uses port 9229).
+
+## Coverage numbers from System Tests
+
+System tests exercise a wide range of modules and, thus, are well-suited for
+providing an upper bound on our coverage numbers. However, they are harder to
```

Comment:
"Upper bound" is the key here. Just loading the instrumented modules brings in a lot of line coverage, even if they are not tested explicitly.